src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1355,6 +1377,111 @@ def _deny_request(
     )
 
 
+@router.post(
+    PRIVACY_REQUEST_PRE_APPROVE_ELIGIBLE,
+    status_code=HTTP_200_OK,
+)
+def mark_privacy_request_pre_approve_eligible(
+    privacy_request_id: str,
+    *,
+    db: Session = Depends(deps.get_db),
+    config_proxy: ConfigProxy = Depends(deps.get_config_proxy),
+    webhook: PreApprovalWebhook = Security(
+        verify_callback_oauth_pre_approval_webhook, scopes=[PRIVACY_REQUEST_REVIEW]
+    ),
+) -> None:
+    """
+    Marks privacy request as eligible for automatic approval.
+    If all webhook responses have been received and all are affirmative, proceed to queue privacy request
+    """
+    PreApprovalWebhookReply.create(
+        db=db,
+        data={
+            "webhook_id": webhook.id,
+            "privacy_request_id": privacy_request_id,
+            "is_eligible": True,
+        },
+    )
+
+    all_webhooks = PreApprovalWebhook.all(db)
+    if not all_webhooks:
```

Comment:
Will this ever get hit? It seems like the verify_callback_oauth_pre_approval_webhook is retrieving the PreApprovalWebhook from the request so we have at least one.

Reply:
true! I can remove